app/src/test/java/org/mozilla/tv/firefox/helpers/FirefoxRobolectricTestRunner.kt

```diff
@@ -0,0 +1,29 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.helpers
+
+import org.mozilla.tv.firefox.helpers.shadows.ShadowADMMessageHandlerBase
+import org.mozilla.tv.firefox.helpers.shadows.ShadowADMMessageReceiver
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+
+class FirefoxRobolectricTestRunner(testClass: Class<*>) : RobolectricTestRunner(testClass) {
+
+    /**
+     * Add ADM class shadows to global robolectric test runner config.
```

Comment:
nit: The first line of this kdoc is redundant to the code. The second line is far from the code where it happens and is likely to not get updated if the code gets updated.

Can we remove the first line and put the second line above the related code?